packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -12,14 +12,27 @@ export const TableVariant: {
   'compact': 'compact'
 }
 
+export interface IRowData {
+}
```

Comment:
I think it might be useful to name this type, because we might add more typed properties to it in the future + consumers can reference it right now.